lib/views/create-bug.js

```diff
@@ -122,6 +116,7 @@ function bugChanged() {
     clearTimeout(timeout);
   }
   timeout = setTimeout(updateBugData, 1000);
```

Comment:
Yeah, you could even rename ```timeout``` to ```debouncer``` and I would immediately know what's going on here, even without a comment.